optaplanner-docs/src/main/docbook/en-US/Chapter-Planner_configuration/Chapter-Planner_configuration.xml

```diff
@@ -436,6 +436,7 @@ public class Queen {
         anyway. For example, a <literal>Queen</literal>'s <literal>row</literal> variable should not be used.</para>
       </section>
     </section>
+<!-- gemma: edited to here -->
```

Comment:
Hey Gemma,

Would you mind not adding these comments in the document itself, but track it externally?
When I change something on the docs with my docbook WYSIWYG editor, this line changes too automatically, so that would create a merge conflict with your changes I fear. 
Also note that we can release a Beta of master at any time: it should always be in a Beta releasable state.

I'll delete this line (if you deleted it locally meanwhile, that's the same change so it probably won't give a merge conflict). If you do get a merge conflict: that can happen, resolve it :)